### ISSUE TITLE
LazyCache not recovering sequences

### ISSUE THREAD

**Issue by PJpb** [drivenbyentropy/aptasuite#62]
Hi,
The LazyCache doesn't work properly - after getting back to a cluster that has been previously analyzed and stored in the cache, the sequences are not displayed at all. Windows PowerShell log attached.
[bug_report.txt](https://github.com/drivenbyentropy/aptasuite/files/2496070/bug_report.txt)

Edit: What is more, no new (not analyzed before and stored in the cache) clusters can be displayed - the loading goes fine, but no sequences are displayed. In both the cases described the charts on the right hand side (e.g. Cardinality Chart) do not display as well.

**Comment by drivenbyentropy** [comment]
Hi, 
Thanks for reporting this. Does his affect only data sets created with previous versions of AptaSuite or also those created with the current one? In addition, could you provide me with the OS and version you are running? If you have the chance, would you be able to see if you can replicate this error on a Linux machine? 

I will see if I can reproduce it locally in the meanwhile.

Thanks again!
Jan

**Comment by PJpb** [comment]
Hi Jan,
OS is Windows 10 Home, version 1803, compilation 17134.345.
I do not have an access to a Linux machine at the moment, I'll see what I can do next week.

This situation happend for the data parsed from scratch and clustered with the v0.9.3.
When I restarted the AptaSuite, I could load a cluster analyzed previously rather quickly (from LazyCache), and I could also load new clusters (not previously analyzed at any timepoint). However, when I open a previously loaded cluster from the cache, I get the same bug as described (
[bug_report2.txt](https://github.com/drivenbyentropy/aptasuite/files/2498389/bug_report2.txt) ). This means that:
- new (not analysed previously) clusters can be loaded
- a previously analysed cluster can be opened only as the first cluster after starting the AptaSuite
- a previously analysed cluster cannot be opened afterwards (i.e. if it's not a "virgin" ClusterAnalysis window ;) )
- the LazyCache doesn't get cleaned after restarting AptaSuite, nor after restarting the OS (tried that too)   

Hope that helps to locate the bug!

**Comment by PJpb** [comment]
Here you go:
```
java version "1.8.0_151"
Java(TM) SE Runtime Environment (build 1.8.0_151-b12)
Java HotSpot(TM) 64-Bit Server VM (build 25.151-b12, mixed mode)
```

**Comment by PJpb** [comment]
It seems that this is a good lead - it worked nicely only when I allowed for all the computations to be done, including cluster cardinality, logo and mutations analysis. I did it a few times with a few clusters and the LazyCache worked well (as long as I waited long enough - it does take a bit of time for my laptop to compute the mutations and cardinality, especially for bigger clusters).
Then I tried to replicate the error by changing the cluster before the additional analysis in the right panel got computed and it did crash in the same way as described above.

Good hunting! ;)
